sentry_sdk/transport.py

```diff
@@ -105,7 +105,7 @@ def _send_event(self, event):
         logger.debug(
             "Sending %s event [%s] to %s project:%s"
             % (
-                event.get("level") or "error",
```

Comment:
it'd make more sense to display the event type and ignore the level

Reply:
Logging both now. Usually there's no event type on our side (we let the server figure it out)